examples/with-auth/index.js

```diff
@@ -0,0 +1,118 @@
+const { setQueues, router, BullMQAdapter } = require('bull-board');
+const { Queue: QueueMQ, Worker, QueueScheduler } = require('bullmq');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const { ensureLoggedIn } = require('connect-ensure-login');
+
+const app = require('express')();
+
+// Configure the local strategy for use by Passport.
+//
+// The local strategy require a `verify` function which receives the credentials
+// (`username` and `password`) submitted by the user.  The function must verify
+// that the password is correct and then invoke `cb` with a user object, which
+// will be set at `req.user` in route handlers after authentication.
+passport.use(new LocalStrategy(
+  function (username, password, cb) {
+    if (username === 'bull' && password === 'board') {
```

Comment:
Should we put it insde the .env file? I think is the best solution at first glance

Reply:
it is just an POC example, do you think that it is better to follow the best practices?

Reply:
Hahah sorry @felixmosh I'm new to reviewing open source projects.

The main idea seems good to me, I think it's simple and I solve our problem.

Reply:
I don't think that there is much "special" here, just a simple passport example (which is copied with the comments from passport's example :] )

Reply:
Ok, good.

Now, we need other Collaborator approve this PR? So, we can finish the implementation? What is the worflow?

Reply:
Just approve the PR it self, then I will be able to merge it, thanx!